app/src/main/res/layout-land/adyen_credit_card_layout.xml

```diff
@@ -116,48 +115,64 @@
           app:layout_constraintBottom_toTopOf="@id/bottom_separator"
           app:layout_constraintEnd_toEndOf="parent"
           app:layout_constraintStart_toStartOf="@id/mid_guideline"
           app:layout_constraintTop_toBottomOf="@id/top_separator"
           >
-        <RelativeLayout
+
+        <ScrollView
+            android:id="@+id/cc_info_view"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_gravity="center"
+            android:layout_height="match_parent"
+            android:overScrollMode="never"
+            android:paddingStart="@dimen/big_margin"
+            android:paddingEnd="@dimen/big_margin"
+            android:scrollbars="none"
             >
-          <include
-              android:id="@+id/adyen_card_form"
-              layout="@layout/selected_payment_method_cc"
+          <RelativeLayout
               android:layout_width="match_parent"
               android:layout_height="wrap_content"
-              />
-          <Button
-              android:id="@+id/change_card_button"
-              style="?android:attr/borderlessButtonStyle"
-              android:layout_width="wrap_content"
-              android:layout_height="@dimen/button_height"
-              android:layout_below="@id/adyen_card_form"
-              android:layout_alignParentEnd="true"
-              android:layout_marginEnd="16dp"
-              android:fontFamily="sans-serif-medium"
-              android:lineSpacingExtra="5sp"
-              android:text="@string/activity_iab_change_card"
-              android:textColor="@color/btn_cancel_text"
-              android:textSize="12sp"
-              android:visibility="gone"
-              />
-        </RelativeLayout>
-      </ScrollView>
+              android:layout_gravity="center"
+              >
+            <include
+                android:id="@+id/adyen_card_form"
+                layout="@layout/selected_payment_method_cc"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                />
+            <Button
+                android:id="@+id/change_card_button"
+                style="?android:attr/borderlessButtonStyle"
+                android:layout_width="wrap_content"
+                android:layout_height="@dimen/button_height"
+                android:layout_below="@id/adyen_card_form"
+                android:layout_alignParentEnd="true"
+                android:layout_marginEnd="16dp"
+                android:fontFamily="sans-serif-medium"
+                android:lineSpacingExtra="5sp"
+                android:text="@string/activity_iab_change_card"
+                android:textColor="@color/btn_cancel_text"
+                android:textSize="12sp"
+                android:visibility="gone"
+                />
+          </RelativeLayout>
+        </ScrollView>
+
+        <include
+            android:id="@+id/fragment_adyen_error"
```

Comment:
Try to add this error view inside the relative layout inside the scroll view instead of creating a new relative layout